### ISSUE TITLE
NativeStorage.keys() is not working in chrome browser...

### ISSUE THREAD

**Issue by mezykr** [TheCocoaProject/cordova-plugin-nativestorage#86]
`NativeStorage.keys(
() => {console.log('success');}, 
() => {console.log('error');}
);`

causes an error:

`Uncaught TypeError: Cannot read property '0' of undefined
    at StorageHandle.LocalStorageHandle [as storageHandlerDelegate] (LocalStorageHandle.js:5)
    at StorageHandle.keys (mainHandle.js:397)
    at <anonymous>:1:15`

**Comment by tomilofano** [comment]
@alokrajiv 
I get the same issue.

My code: 
```
public inicialite():Promise<boolean>{
    return this.nativeStorage.setItem("hola", "hola").then(() => {
      return this.nativeStorage.setItem("chau", "chau").then(() => {
        return this.nativeStorage.getItem("hola").then((ho) => {
          alert(JSON.stringify(ho));
        return this.nativeStorage.keys().then((llaves) => {
          alert(JSON.stringify(llaves));
          return true;
          }).catch((irr) => {alert("err 4 " + irr)});
        }).catch((irr) => {alert("err 3 " + irr)});
      }).catch((arr) => {alert("err 2 " + arr)});
    }).catch((err) => {alert("err 1 " + err)});
  }
```

the outpus:
```
"hola" 

err 4 TypeError: Cannot read property '0' of undefined
```

Command: 
```
$ionic cordova run browser
```

Ionic info:
```
global packages:

    @ionic/cli-utils : 1.4.0
    Cordova CLI      : 7.0.1 
    Ionic CLI        : 3.4.0

local packages:

    @ionic/app-scripts : 2.0.2
    Cordova Platforms  : browser 4.1.0
    Ionic Framework    : ionic-angular 3.5.3

System:

    Node       : v6.10.2
    OS         : Linux 4.8
    Xcode      : not installed
    ios-deploy : not installed
    ios-sim    : not installed
    npm        : 3.10.10 
```

Google chrome version:
```
Version 58.0.3029.110 (64-bit)
```

**Comment by alokrajiv** [comment]
Sorry...guys. Let me look into it. Was a bit busy! :) 

There is a PR #85 that I think will fix. I'll pull it tonight if it passes all test. Thanks!

**Comment by alokrajiv** [comment]
I have merged the branch. Sorry for the delay again guys.

@mezykr @tomilofano could you please confirm if this issue is fixed now. 

PS: This version is not yet released to npm. Please install from master

`cordova plugin add https://github.com/TheCocoaProject/cordova-plugin-nativestorage`

**Comment by alokrajiv** [comment]
I have merged PR #85 . Sorry for the delay again guys.

@mezykr @tomilofano I have recreated your error and this has fixed the issue.

Thanks for the reporting!!

For anyone wanting to solve this before we make the new release:

Please install directly from GitHub master branch:

`cordova plugin add https://github.com/TheCocoaProject/cordova-plugin-nativestorage`

cc @GillesC (we have a few other issue. lets probably release after fixing more issues.)
